index.html

```diff
@@ -429,19 +430,20 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
     }
 
     if (unsupported.os) {
-        if (!unsupported.default) {
-            unsupported.default = "Unfortunately, we don't support " + unsupported.os + " any longer. <a href='" + unsupported.url + "'>Take a look at the Requirements.</a>";
+        if (!unsupported.defaultText) {
+            unsupported.defaultText = "Unfortunately, we don't support " + unsupported.os + " any longer. <a href='" + unsupported.url + "'>Take a look at the Requirements.</a>";
```

Comment:
Linking to the requirements doesn't make much sense. The requirements don't list and reasons why they are the requirements - and as such, don't help someone understand why support for and OS was dropped. I would expect that any such dropping of support would have a blog post (similar to the Windows XP blog post), and would expect a link to that instead of the requirements.

Perhaps linking to the requirements is a good idea both here and in the Windows XP unsupported section.